{"name": "test_equality_failure", "status": "failed", "statusDetails": {"message": "AssertionError: assert 10 == 11", "trace": "@allure.feature('test_failure')\n    def test_equality_failure():\n>      assert 10 == 11\nE      assert 10 == 11\n\ntest_fail.py:18: AssertionError"}, "start": 1692071014230, "stop": 1692071014230, "uuid": "da2c1b7f-53aa-4988-98f6-d0167806d142", "historyId": "09c67c1678f777b887b7a82c4334233e", "testCaseId": "09c67c1678f777b887b7a82c4334233e", "fullName": "test_fail#test_equality_failure", "labels": [{"name": "feature", "value": "test_failure"}, {"name": "suite", "value": "test_fail"}, {"name": "host", "value": "whtdeMacBook-Pro.local"}, {"name": "thread", "value": "66349-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_fail"}]}